### ISSUE TITLE
setLowValue(max) doesn't work

### ISSUE THREAD

**Issue by bnlokash** [githuboftigran/rn-range-slider#14]
```
<Slider
  ref={ component => this._sliderRef = component }
  rangeEnabled={false}
  min={1}
  max={5}
  step={1}
/>

this._sliderRef.setLowValue(5)
```

Calling `setLowValue` with the max value causes the slider to render at the 4th position, not the 5th. 
Calling `setLowValue` with 1, 2, 3, or 4 sets the slider position correctly.
Tested on iOS simulator

**Comment by bnlokash** [comment]
Calling `setLowValue` with 1, 2, 3, or 4 sets the slider position correctly

**Comment by githuboftigran** [comment]
@bnlokash yes, because low value can't be equal to high value and high value can't be higher than max

**Comment by bnlokash** [comment]
I'm not sure I follow. In my usage there is only one slider, which I assume is the low. How would I set this to the max?

**Comment by githuboftigran** [comment]
Aaa, right... Will fix in near future, thanks for reporting.

**Comment by githuboftigran** [comment]
@bnlokash , fixed in 1.2.5
